test/ref_ops_test.cpp

```diff
@@ -848,11 +848,42 @@ TEST_CASE(contiguous_test)
     p.compile(migraphx::ref::target{});
     auto result = p.eval({}).back();
 
+    result.visit([&](auto output) {
+        std::vector<size_t> new_strides = {12, 4, 2, 1};
+        EXPECT(bool{output.get_shape().strides() == new_strides});
```

Comment:
Why do you do a visit here? Why not just `EXPECT(result.get_shape().strides() == new_strides)`?